apps/frontend/src/components/global/upload_tabs/splunk/AuthStep.vue

```diff
@@ -84,39 +88,47 @@ export default class AuthStep extends Vue {
   hostname = '';
   index = '';
 
+  /** Form required field rules. Maybe eventually expand to other stuff */
+  reqRule = (v: string | null | undefined) =>
+    (v ?? '').trim().length > 0 || 'Field is Required';
+
   async login(): Promise<void> {
-    if (!/^https?:\/\//.test(this.hostname)) {
-      this.hostname = `https://${this.hostname}`;
-    }
+    if (this.index !== '') {
```

Comment:
do this as a guard clause, i.e. if this.index === '', throw that error.  then you can have the rest of your code be unindented

Reply:
Maybe I'm missing something, but how would error handling work this high up in the frontend?

Reply:
What Amndeep is saying is use:
if (this.index !== '') {
    SnackbarModule.failure('Failed to login - A valid index is required');
} else {
   ....
}

Reply:
don't even need an explicit else clause if you just return early from the function

Reply:
Issue should be addressed now